Makefile

```
@@ -115,7 +164,8 @@ build: build-linux-amd build-linux-arm build-macos-intel build-macos-apple build
 
 #@ Lint
 define run-lint
-	@if command -v ${1} &> /dev/null; then \
+	@export PATH="$${PATH}:$${PWD}/.bin"; \
+	if command -v ${1} > /dev/null; then \
```

```suggestion
	if command -v ${1} &> /dev/null; then \
```

As per previous comment, github says this was applied, am I crazy?